modules/instrument_builder/js/instrument_builder.instrument.js

```diff
@@ -89,6 +89,9 @@ var Instrument = {
             }
             for (element of elements[i].Elements) {
                 switch (element.Type) {
+                    case "line":
```

Comment:
there is no "line" type saved in a linst file. Why did you add this type? It's always been a static label with a <br> content.

Reply:
@driusan I removed it. I've tested the code and that part is unnecessary. I added it because I thought it was necessary from reviewing @kongtiaowang's past PR on this issue.